js/views/harvester.vue

```diff
@@ -111,6 +122,15 @@ export default {
         }
     },
     methods: {
+        run() {
+            this.source.run(() => {
+                this.$dispatch('notify', {
+                    autoclose: true,
+                    title: this._('Run requested'),
+                    details: this._('The run will soon appear in the list. You will need to reload the page to see it.')
```

Comment:
You can extract translations before merging :)

Reply:
Done in 56968453f554d780ca013438f4cf672b88577801